test/e2e/apps/daemon_set.go

```diff
@@ -888,20 +889,20 @@ func clearDaemonSetNodeLabels(c clientset.Interface) error {
 func updateNamespaceAnnotations(c clientset.Interface, nsName string) (*v1.Namespace, error) {
```

Comment:
if we go from update->patch, do we want to rename this method to reflect that?

Looks like it's called from only one spot:
https://cs.k8s.io/?q=updateNamespaceAnnotations&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
Makes sense. Updated it, thank you.